assets/js/image_expansion.js

```diff
@@ -86,10 +86,15 @@ function pickAndResize(elem) {
     clearEl(elem);
   }
 
+  let muted = 'muted';
+  if (store.get('unmute_videos')) {
+    muted = '';
+  }
```

Comment:
would a ternary be better? e.g. `const muted = store.get('unmute_videos') ? '' : 'muted';`